backend/src/main/kotlin/xyz/elitese/ehrenamtskarte/importer/general/ImportToDatabase.kt

```diff
@@ -0,0 +1,57 @@
+package xyz.elitese.ehrenamtskarte.importer.general
+
+import com.beust.klaxon.Klaxon
+import org.jetbrains.exposed.dao.id.EntityID
+import org.jetbrains.exposed.sql.transactions.transaction
+import org.postgis.Point
+import xyz.elitese.ehrenamtskarte.database.*
+import xyz.elitese.ehrenamtskarte.importer.freinet.AcceptingStoresImporter
+import xyz.elitese.ehrenamtskarte.importer.freinet.types.Category
+
+object ImportToDatabase {
+
+    fun prepareCategories() {
+        val categoriesJson = AcceptingStoresImporter::class.java
+                .getResource("/freinet_import/categories.json").readText()
+        val categories = Klaxon().parseArray<Category>(categoriesJson)!!
+
+        transaction {
+            categories.forEach { CategoryEntity.new(it.id) { name = it.name } }
+        }
+    }
```

Comment:
why is this part of `general` as this is freinet related?

Reply:
It isn't